master_changes.md

```diff
@@ -3,6 +3,10 @@ note.
 Possibly scripts breaking changes are prefixed with ✘.
 New option/command/subcommand are prefixed with ◈.
 
+## Global CLI
+  * --help/--version documented in wrong section for aliases
+  * opam lock --help missing common information
```

Comment:
```suggestion
  * --help/--version documented in wrong section for aliases [#4317 @dra27]
  * opam lock --help missing common information [#4317 @dra27]
  * Use `Cmdliner.Manpage.s_*` instead of constants  [#4317 @dra27]
```

Reply:
Oops - updated!